htdocs/xoops_lib/Xoops/Core/Logger.php

```diff
@@ -204,7 +204,7 @@ private function reportFatalError($msg)
         if ('cli' === php_sapi_name()) {
             fprintf(STDERR, "\nError : %s\n", $msg);
         } else {
-            printf(_XOOPS_FATAL_MESSAGE, XOOPS_URL, $msg);
+            printf(_XOOPS_FATAL_MESSAGE, XOOPS_URL, utf8_encode($msg));
```

Comment:
The error message "Could not connect to database..." is translated to portuguese in my browser although I don't have the portuguese translation files on Xoops. I'm using English but somehow that error message comes translated and it is not utf-8 encoded. Do you know where that message comes from?